frontend/src/components/options/seasons/useSelectedSeasonsState.ts

```diff
@@ -3,29 +3,22 @@ import { useSearchParams } from 'react-router';
 export function useSelectedSeasonsState() {
```

Comment:
Do not forget to delete this file and the `SelectedComparisionSeasonos.tsx` file. They are no longer used.